Sources/DistributedActors/Serialization/Serialization+Invocation.swift

```diff
@@ -21,14 +21,20 @@ import SwiftProtobuf
 
 public struct ClusterInvocationEncoder: DistributedTargetInvocationEncoder {
     public typealias SerializationRequirement = any Codable
-    var arguments: [Data] = []
+    public internal(set) var arguments: [Data] = []
```

Comment:
Hm this is because the cross module of the Singleton... I don't love exposing this as "really public" API... Need to think how to manage our cross module things. :-/

Reply:
specifically about this Data though we should be able to not serialize at all when the message is still stashed, OR may be delivered locally -- so we don';t do a serialization roundtrip... might need to have some two "modes" enum in the encoder.

I wish we were able to have just "different encoder per call" but we can't because compiler limitations: the ad-hoc protocol requirements strike again 😞 So we'll have to have one encoder type with "modes"